### DISCUSSION TITLE
Github to vercel deployment, all check failed

### DISCUSSION THREAD

**Discussion by zafar-saleem** [vercel/community#3616] [Help]
### Summary

When I push changes to all of my github repo. It is supposed to trigger deployments(it was triggering before). But the last few commits is showing me error "All checks failed".

### Steps to Reproduce

When I commit new commits to my github repo it is showing me below error.

<img width="544" alt="Screenshot 2023-08-15 at 12 56 44 PM" src="https://github.com/vercel/community/assets/1269344/902e2d38-fd8d-4ee0-8b04-09c5858f9b81">

All of my commits were triggering deployment however, last few commits started to show my above error. Any idea what could have possibly gone wrong?

BTW I was making too many commits since morning in quick succession. I checked the limitation where I could not find any per day or per hour limitations on vercel docs.

**Comment by amyegan** [comment]
Hey @zafar-saleem. This is likely due to a build failure. Please check the Vercel project dashboard to get more information about what went wrong.

**Reply by zafar-saleem** [reply]
@amyegan I only see the successful deployments in vercel dashboard. These failed ones do not somehow get displayed in vercel dashboard deployments tab. When I build it locally it gets built successfully though.

**Reply by amyegan** [reply]
Do you have Ignored Build Step configured or [`git.deploymentEnabled`](https://vercel.com/docs/projects/project-configuration/git-configuration#git.deploymentenabled) set to `false`?

**Comment by stephanbogner** [comment]
I have the same issue:
The build is successful locally, however when pushed it says "All checks failed". This only shows up in Github without any explanation and nothing in the Vercel Dashboard at all.
(Neither Ignored Build Step or deploymentEnabled configured at all)

**Reply by amyegan** [reply]
Do you have any additional checks configured in GitHub?

**Reply by stephanbogner** [reply]
Where can I check this?
(Which also means, likely no, I only connected via Vercel and didn't configure anything in Github)

**Reply by stephanbogner** [reply]
Very weird – I now tried rebuilding a few times and now the Github popover shows a "Details"-button and I can see logs in the Vercel dashboard (which say `The file "/vercel/path0/.next/routes-manifest.json" couldn't be found. This is often caused by a misconfiguration in your project.` :tada: )

**Reply by stephanbogner** [reply]
And now the deploy works again ... but I don't understand why :sweat_smile:

**Comment by nthnwlkr** [comment]
I finally figured this out, had the same issue, commits in Github would show the "All checks failed" error message and no associated build would be visible in Vercel. 

Using the `vercel` command to deploy via the CLI I got this message `Error: Resource is limited - try again in 2 hours (more than 100, code: "api-deployments-free-per-day")` and so it turned out that I'd overrun the [100 build daily rate limit](https://vercel.com/docs/limits/overview#general-limits) for my Vercel Hobby plan. Since the rate limit renews every day, this is possibly why builds appear to magically start working again for some people after a while. 

It would be great if the Vercel UI could indicate this is happening, and/or for the Github Vercel app "all checks failed" message to give more context.

**Reply by benjamincarp** [reply]
I also seem to be hitting this and it was very frustrating to have no indication in the dashboard. I definitely wasted over an hour trying to troubleshoot this before turning to the CLI to finally get the error.

**Reply by thathurtabit** [reply]
Thanks for this, in my case, it was down to the `cron` schedule being invalid - I couldn't get any info on this via my github commit, and only trying to deploy via `vercel cli` gave me the info I needed to fix the issue.

// Via vercel cli
```
Error: Error while validating your Cron Jobs expressions:
```

**Reply by stephanbogner** [reply]
This would explain how I perceive the behaviour. It would be great if Vercel could show this error also on Github/Vercel.
I commonly develop on hobby until the project is migrated to the client (which uses a paid tier). If I would get the above error I would at least know what's going on.

**Reply by PanditSiddharth** [reply]
I also have same issues
![Screenshot from 2024-02-01 14-00-29](https://github.com/vercel/community/assets/75090218/34166e49-8cdf-453a-bb79-c485578fb86b)

**Reply by RudrThakur** [reply]
for me it was invalid format of the cron expression, as I was deploying a NESTJS application which had a different cron expression. I thght it would fix the issue but then, this happened: 

`Error: Hobby accounts are limited to daily cron jobs. This cron expression (*/1 * * * *) would run more than once per day. Upgrade to pro to unlock all Cron Jobs features on Vercel.`

So, turns out you have to pay for cron jobs that runs more than once a day. Sad.

**Reply by Anoa2024** [reply]
Than the cronjobs in hobby-plan are totally useless. Ok I will update my data out of vercel and do it with an backend
